RELEASE.md

```diff
@@ -93,6 +93,8 @@ Also the images we build are based on some image specified in the `FROM` stateme
     git push --follow-tags <upstream> main
     ```
 
+  - [ ] Update baseVersion in chartpress to the next release (e.g. `2.1.0-0.dev` after 2.0.0)
+
   - [ ] Create a GitHub release.
```

Comment:
@consideRatio do you know why we have 'make a GitHub Release' in our release process? To me, those are for projects that produce artifacts to attach, and/or use GitHub Releases for release notes, and we do neither of these. I'm not sure what's beneficial in what we do beyond what's already in the autogenerated releases for tags.

Reply:
Yeah I've added it because when projects once has made them (they are for z2jh), they show up in the web UI. So, if someone makes 2.0.0 and not make a github release, it will show up as 1.2.0 there as the latest version.

So, it is a precaution. I'd wish we could make github stop presenting that at all for us really though.

Reply:
Should we remove them all? I'll open an issue about it.

Reply:
Yeah, I really wish it were easier (or possible) to hide unused features on GitHub. Should we delete all the releases, then? Or just edit the latest release description to point to where actual releases are published?

Reply:
Opened #2861 about it.